lightspeed/modules/proc_update-redirect-uri.adoc

```diff
@@ -0,0 +1,70 @@
+:_content-type: PROCEDURE
+
+[id="update-redirect-uri_{context}"]
+
+= Updating the Redirect URIs
+
+When Ansible users log in or log out of Ansible Lightspeed service, the {PlatformName} redirects the user's browser to a specified URL. You must configure the redirect URLs so that users can log in and log out of the application successfully.  
+
+.Prerequisites
+* You have created and deployed a {LightspeedShortName} instance to your namespace.
+
+.Procedure
+. Get the URL of the Ansible Lightspeed instance:
+.. Go to {OCP}.
+.. Select *Networking → Routess*.
+.. Locate the {LightspeedShortName} instance that you created. 
+.. Copy the Location URL of the {LightspeedShortName} instance.
+
+. Update the *login* redirect URI:
+.. In the automation controller, go to *Administration → Applications*.
+.. Select the Lightspeed Oauth application that you created.
+.. In the *Redirect URIs* field of the Oauth application, enter the URL in the following format:
++
+`https://<lightspeed_route>/complete/aap/`
++
+An example of the URL is `https://lightspeed-on-prem-web-service.com/complete/aap/`.
++
+[IMPORTANT]
+====
+The Redirect URL must include the following information:
+
+* The prefix `https://`
+* The `<lightspeed_route>` URL, which is the URL of the {LightspeedShortName} instance that you copied earlier
+* The suffix `/complete/aap/` that includes a backslash sign (*/*) at the end
+====
+.. Click *Save*. 
+
+. Update the *logout* redirect URI:
+.. Log in to the cluster on which the {PlatformName} instance is running. 
+.. Identify the *AutomationController* custom resource.
```

Comment:
Is there a step missing here? Does the user have to do something after identifying the custom resource to be able to edit the YAML? Disregard if not relevant.

Reply:
Great catch! Added the step 'Select YAML view' after 'identify the automation controller custom resource'.